src/Apps/Artist/Routes/Overview/Components/ArtistRecommendations.tsx

```diff
@@ -0,0 +1,57 @@
+import { Serif } from "@artsy/palette"
+import { ArtistRecommendations_artist } from "__generated__/ArtistRecommendations_artist.graphql"
+import { ArtistRecommendationsQuery } from "__generated__/ArtistRecommendationsQuery.graphql"
+import { Overview_artist } from "__generated__/Overview_artist.graphql"
+import { SystemContext } from "Artsy"
+import { renderWithLoadProgress } from "Artsy/Relay/renderWithLoadProgress"
+import React, { useContext } from "react"
+import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+
+interface ArtistRecommendationsProps {
+  artist: ArtistRecommendations_artist
+}
+
+export const ArtistRecommendations: React.FC<
+  ArtistRecommendationsProps
+> = props => {
+  const {
+    artist: { name },
+  } = props
+
+  return (
+    <div>
+      <Serif size="8" color="black100">
+        Related to {name}
+      </Serif>
+    </div>
+  )
+}
+
+export const ArtistRecommendationsFragmentContainer = createFragmentContainer(
+  ArtistRecommendations,
+  graphql`
+    fragment ArtistRecommendations_artist on Artist {
+      name
+    }
+  `
+)
+
+export const ArtistRecommendationsQueryRenderer: React.FC<{
+  artist: Overview_artist
+}> = ({ artist }) => {
+  const { relayEnvironment } = useContext(SystemContext)
```

Comment:
You should be able to also destructure `user` here as well (and have lab feature access). What do you think about moving the `showRecommendations && ...` logic [from here](https://github.com/artsy/reaction/pull/2236/files#diff-bfc57d8ce9d60a71675e59322d85c6f3R182) into this query renderer so that it's all just in one place (`return null` if the user doesn't have access).

It might be easier to then keep track/change that condition later.

Reply:
Yeah, good call! I will update that.

Reply:
Looking into this, I prefer it where it's at. The `<ArtistRecommendations>` component [gets wrapped in a `<Row><Col>`](https://github.com/artsy/reaction/pull/2236/files#diff-bfc57d8ce9d60a71675e59322d85c6f3R182). I'd rather the Row/Col be in the `Overview` than in the `ArtistRecommendations`, and I don't think they should be emitted unless the recommendations are going to be emitted.

Reply:
Cool, makes sense!